src/Sarif.Driver/Sdk/AnalyzeCommandBase.cs

```diff
@@ -277,8 +286,17 @@ protected virtual void InitializeConfiguration(TOptions options, TContext contex
         {
             context.Policy = new PropertiesDictionary();
 
-            if (String.IsNullOrEmpty(options.ConfigurationFilePath) ||
-                options.ConfigurationFilePath == DefaultPolicyName)
+            if (options.ConfigurationFilePath == DefaultPolicyName)
+            {
+                return;
+            }
+
+            if (String.IsNullOrEmpty(options.ConfigurationFilePath) &&
+                    File.Exists(DefaultConfigurationPath))
+            {
+                options.ConfigurationFilePath = DefaultConfigurationPath;
+            }
+            else
```

Comment:
NIT, considering reversing this conditional to that exit occurs here and then the positive code path happens as follow-on execution. i.e., 

if (!String.IsNullOrEmpty(options.blah) ||
    !File.Exists(blah)
{
  return;
}

options.ConfigurationFilePath = DefaultConfdigblah..